PostProcessing/Runtime/PostProcessRenderContext.cs

```diff
@@ -380,8 +392,12 @@ RenderTextureDescriptor GetDescriptor(int depthBufferBits = 0, RenderTextureForm
             //intermediates in VR are unchanged
             if (stereoActive && desc.dimension == Rendering.TextureDimension.Tex2DArray)
                desc.dimension = Rendering.TextureDimension.Tex2D;
-          
+            
+#if UNITY_2019_1_OR_NEWER
```

Comment:
#if logic is reversed

Reply:
Is it? Pretty sure this is correct. RenderTextureDescriptor didn't get the useDynamicScale parameter until 2019.1

Reply:
Ok, my fault. You're right.